src/targets/web/globals.js

```diff
@@ -1,61 +1,12 @@
+import memoize from '@emotion/memoize'
```

Comment:
This one is widely used https://github.com/alexreardon/memoize-one

Reply:
the use case is completely different though - `memoize-one` only memoizes the last call (and replaces the cached values when new arguments get passed in), whereas `@emotion/memoize` keeps the cached results for each argument

Reply:
My only worry is, this thing is getting executed a lot, and it used to be a simple look up table with immediate access. Especially when the animation starts it needs to run as fast as possible in order to counteract frame skipping. Are you sure we need memoized runtime regexp's for tree shaking or can't this be solved in any other way?

Reply:
Memoization effectively turns this into a lookup table (once the memoized cache gets populated). The same technique is used by emotion which is perf-concerned css-in-js & is not causing any bottleneck for us there. One thing I'm sure is that RegExp + memoize is better than just RegExp.

The other solution to make this tree-shakeable would be to rewrite lookup table creation slightly so the lookup table doesnt get manipulated in unobvious way with forEach, to put in other words - probably some reduce would be sufficient to make this happen. Folding this into a RegExp rather than a lookup table saved few bytes (100~ when I was testing this in isolation) that's why I've decided to use this technique instead. Can "revert" to reduce approach if u want, just let me know

Reply:
I think i'd be happier with a reduce, if you could do that it would be great!